### PR TITLE
Create a token for programmatic access for cnf-ci

### PR DESCRIPTION
**Pull request by sabinaaledort**

### CODE REVIEW

**Comment by sabinaaledort** [comment]
@alvaroaleman can you take a look please?

**Review by alvaroaleman — APPROVED** [review]

File: clusters/app.ci/registry-access/cnf-ci/README.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+# EDGE Infrastructure
```
Comment (yuvalk) [inline clusters/app.ci/registry-access/cnf-ci/README.md:1]: we're not edge 
better make it "Telco 5G"

File: clusters/app.ci/registry-access/cnf-ci/README.md [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# Telco 5G Infrastructure
+
+## Generating an Image Pull Credential
+
+First, log in to [the cluster](https://console-openshift-console.apps.ci.l2s4.p1.openshiftapps.com/topology/all-namespaces/graph). Then, run:
+
+```sh
+oc get secrets --namespace edge -o json | jq '.items[] | select(.type=="kubernetes.io/dockercfg") | select(.metadata.annotations["kubernetes.io/service-account.name"]=="image-puller") | .data[".dockercfg"]' --raw-output | base64 --decode | jq
```
Comment (yuvalk) [inline clusters/app.ci/registry-access/cnf-ci/README.md:8]: that's the wrong namespace

**Comment by sabinaaledort** [comment]
@alvaroaleman can you take a look again please? i had to update the README

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**